src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -1153,12 +1344,31 @@ extern "C" void sl_bt_on_event(sl_bt_msg_t * evt)
     }
     break;
 
-    default:
+    default: {
         ChipLogProgress(DeviceLayer, "evt_UNKNOWN id = %08" PRIx32, SL_BT_MSG_ID(evt->header));
         break;
     }
+    }
 
+    // Unlock the stack
     chip::DeviceLayer::PlatformMgr().UnlockChipStack();
 }
 
+} // namespace Internal
+} // namespace DeviceLayer
+} // namespace chip
+
+extern "C" void zigbe_bt_on_event(volatile sl_bt_msg_t * evt);
```

Comment:
does this need a #ifdef SL_TEST_MATTER_ZIGBEE_DMP too

Reply:
I'll add one just in case, but since it doesn't get calledI think the linker wouldn't complain about not finding the definition.